Sources/FormattingHelpers.swift

```diff
@@ -1859,6 +1859,7 @@ extension Formatter {
         case staticPropertyWithBody
         case classPropertyWithBody
         case overriddenProperty
+        case swiftUIDynamicProperty
```

Comment:
Instead of coupling this to SwiftUI, what if we made it more general:

```suggestion
        case instancePropertyWithPropertyWrapper
```